src/lnd-setup/generate-lightning-client.js

```diff
@@ -23,7 +23,7 @@ const { ENGINE_STATUSES } = require('../constants')
  * @returns {grpc.Client} lnrpc Lightning client definition
  */
 function generateLightningClient ({ host, protoPath, tlsCertPath, macaroonPath, status, logger }) {
-  const { lnrpc } = loadProto(protoPath)
+  const { lnrpc, invoicesrpc } = loadProto(protoPath, ['rpc.proto', 'invoicesrpc/invoices.proto'])
```

Comment:
is there any way we can make these path strings variables? Either as constants or passed in from the caller? Feels a little weird as is.

Reply:
Updated